pennylane/transforms/qcut.py

```diff
@@ -428,9 +428,10 @@ def graph_to_tape(graph: MultiDiGraph) -> QuantumTape:
     >>> graph = qml.transforms.qcut.tape_to_graph(tape)
     >>> tape = qml.transforms.qcut.graph_to_tape(graph)
     >>> print(tape.draw())
-     0: ──RX(0.4)──────╭C───────────────╭X──┤ ⟨Z⟩
-     1: ──RY(0.5)──────╰X──MeasureNode──│───┤
-     2: ──PrepareNode───────────────────╰C──┤
+    0: ──RX──────────╭●──────────────╭X─┤  <Z>
+    1: ──RY──────────╰X──MeasureNode─│──┤
```

Comment:
I'm curious, why are the phases omitted in such a case?

Reply:
This documentation must have been written prior to the the new `qml.drawer.tape_text` method being called by `tape.draw`.  `qml.drawer.tape_text`, and thus `tape.draw`, omit parameters by default.  They must be requested by `decimals=2`.

Reply:
Nice! Thank you for the clarification.